### PR TITLE
Update Hindi.csv

### PR DESCRIPTION
**Pull request by vajradevam**
some more words

### CODE REVIEW

File: src/data/Hindi.csv [hunk 1]
```diff
@@ -13,27 +14,35 @@ Ajeeb,Nirala
...
 Andaza,Anuman
-Aasaan,Saral
 Asar,Prabhaav
 Aulaad,Santaan
 Aurat,Mahila
 Aurat,Stree
 Azaadi,Swatantrata
+Baazu,Bhuja
+Bachcha,Baalak
+Badbu,Durgandh
```
Comment (venusuranus) [inline src/data/Hindi.csv:28]: Badboo

File: src/data/Hindi.csv [hunk 2]
```diff
@@ -13,27 +14,35 @@ Ajeeb,Nirala
...
+Badtameeji,Dhristata
```
Comment (venusuranus) [inline src/data/Hindi.csv:29]: Badtameezi

File: src/data/Hindi.csv [hunk 3]
```diff
@@ -89,29 +108,44 @@ Intezaar,Pratiksha
...
 Karz,Roon
 Karz,Udhaar
 Khaandaan,Gharana
 Khaasiyat,Visheshta
+Khali,Rikt
 Khana,Bhojan
 Khandaan,Vansh
+Kharab,Akushal
+Khareedna,Molna
+Khatarnaak,Bhayaprad
```
Comment (venusuranus) [inline src/data/Hindi.csv:140]: Bhayanak ?

File: src/data/Hindi.csv [hunk 4]
```diff
@@ -89,29 +108,44 @@ Intezaar,Pratiksha
...
+Khatarnaak,Sankatmay
+Khatra,Sankat
 Khilaaf,Viruddh
 Khof,Bhay
 Khoobi,Visheshta
 Khoon,Rakh
+Khusbu,Sugandh
```
Comment (venusuranus) [inline src/data/Hindi.csv:147]: Khushboo

File: src/data/Hindi.csv (hunk 1 above)
Comment (arnav-db) [inline src/data/Hindi.csv:28]:
```suggestion
Badboo,Durgandh
```

File: src/data/Hindi.csv (hunk 2 above)
Comment (arnav-db) [inline src/data/Hindi.csv:29]:
```suggestion
Badtameezi,Dhristata
```

File: src/data/Hindi.csv (hunk 3 above)
Comment (arnav-db) [inline src/data/Hindi.csv:140]:
```suggestion
Khatarnaak,Bhayanak
```

File: src/data/Hindi.csv (hunk 4 above)
Comment (arnav-db) [inline src/data/Hindi.csv:147]:
```suggestion
Khusboo,Sugandh
```

**Review by arnav-db — APPROVED** [review]

**Comment by arnav-db** [comment]
Thanks for adding all these words 😄

**Comment by vajradevam** [comment]
bhaiya aap discord pe hai kya bahut baat karna tha r/chodi ke ban ke baad

**Comment by arnav-db** [comment]
I don't use discord or reddit now.

**Pull request merged**